.env.staging

```diff
@@ -67,5 +67,11 @@ THUMBNAIL_SERVICE_API_BASE_URL="define THUMBNAIL_SERVICE_API_BASE_URL"
 # Patron Keycloak migration
 PATRON_UPGRADE_URL="define PATRON_UPGRADE_URL"
 
+# 2FA registration
+EMAIL_2FA_REGISTRATION_URL="define EMAIL_2FA_REGISTRATION_URL"
+EMAIL_2FA_REGISTRATION_REDIRECT_URL="define EMAIL_2FA_REGISTRATION_REDIRECT_URL"
+EMAIL_2FA_ACTION="define EMAIL_2FA_ACTION"
+ABOUT_2FA_URL=https://www.nla.gov.au/using-library/getting-started/getting-started-catalogue/two-factor-authentication
+
```

Comment:
does .env.test need update as well

Reply:
Nah, those values are stubbed out in the tests that require them, since they will be testing for their existence also.